glTF-Toolkit/src/GLTFTextureCompressionUtils.cpp

```diff
@@ -261,7 +260,6 @@ void GLTFTextureCompressionUtils::CompressImage(DirectX::ScratchImage& image, Te
         break;
     case TextureCompression::BC7_SRGB:
         compressionFormat = DXGI_FORMAT_BC7_UNORM_SRGB;
-        compressionFlags |= DirectX::TEX_COMPRESS_SRGB_IN;

```

Comment:
why?

Reply:
Because I changed to handle the SRGB conversion during load. Math needs to be done on linear values so I just made it the default.